src/components/views/visualiseView/Visualize.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@mui/styles';
+import MainLayout from '../../../layout/visualise/main';
+import Sidebar from './sidebar/sidebar';
+import { properties } from './sidebar/mock/properties';
+import VisualizeLayout from './Layout';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+}));
+
+const MainVisualize = ({ children }) => {
+  const classes = useStyles();
+  return (
+    <MainLayout sidebar={<Sidebar properties={properties} />}>
+      <VisualizeLayout />
+    </MainLayout>
+  );
+};
+
+export default MainVisualize;
```

Comment:
Why rename this? if needed could we please rename the component as well so we keep consistency whenever possible? Thx